src/components/app-registration/AppRegistration/AppRegistrationForm/AppRegistrationForm.tsx

```diff
@@ -65,7 +65,11 @@ export default function AppRegistrationForm() {
                     <fieldset>
                         <div className={styles.formHeaderContainer}>
                             <h4 className={styles.registerFormHeader}>App information</h4>
-                            <div className={styles.description}>Paste your API token with the admin scope here.</div>
+                            <div className={styles.description}>
+                                <span>
+                                    Paste your API token with the admin scope here.
```

Comment:
@hubert-deriv could you please translate it?

Reply:
Translations is something that is not implemented yet in this project. It is something that is on our roadmap.